SitePackage.lua

```diff
@@ -44,15 +44,25 @@ end
 local function old_module_check(modT)
     -- Check if a module is 'old'
     -- modT should have a entry 'fn' with the module path
-    -- returns true/false
+    -- returns 4 values: true/false, cutoff1, true/false, cutoff2
```

Comment:
looking at this, I would simply bit to a function that returns the 'age' of the module in terms of halve years?

Convert `a` to Jan 1 and `b` to July 1 and substract with the current timestamp divide by 6 months?

The the logic is simply: if age > 7 else if age > 5 ?

Reply:
you still have to calculate the cutoff toolchain generation, so this does not really help.
I've updated it to use timestamps, as that makes it more general indeed.

Reply:
Hmm, I still find it very complex code for something that (at least in my mind) should be simple. Let me think about it for a moment.

Is it worthwhile to keep this cutoff thing in the message? We want them to preferably use the newest toolchain?

Reply:
to me it's not complex at all, just a lot of boilerplate.

but yeah it makes sense to remove the cutoff toolchain generation from the message.